referrer-policy/no-referrer-policy/no-referrer-policy.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE HTML>
+<html>
+  <head>
+    <title>Check that subresource gets the referrer URL when no explicit
+           Referrer Policy is set.</title>
+    <script src='/resources/testharness.js'></script>
+    <script src='/resources/testharnessreport.js'></script>
+    <!-- Common global functions for referrer-policy tests. -->
+    <script src="../generic/common.js"></script>
+  </head>
+  <body>
+    <h1>Check that subresource gets the referrer URL when no explicit Referrer
+        Policy is set.</h1>
+
+    <p>This page loads the subresource and waits for the message containing the
+       referrer URL from the child.</p>
+
+    <script>
+
+      // Display this page's location.
+      appendPageInfo("Parent frame");
+
+      // Embed the iframe on a different domain - load subresource.
+      var iframe = document.createElement('iframe');
+      iframe.type = 'text/javascript';
+      iframe.src = "http://www1." + location.hostname + ":" + location.port +
```

Comment:
Yes. This is explicitly to test cross-origin referrer. Should I point it out in a comment?